bridgetown-website/src/_docs/plugins/html-inspectors.md

```diff
@@ -0,0 +1,65 @@
+---
+title: HTML Inspectors
+order: 0
+top_section: Configuration
+category: plugins
+---
+
+The HTML Inspectors API, added in Bridgetown 1.1, provides a useful and safe way to manipulate the HTML output of your resources. Safe because instead of using string manipulation, regular expressions, and the like—which is prone to error—you'll be working on real node trees. This is thanks to [Nokogiri](https://nokogiri.org), a Ruby gem which lets you work with a DOM-like API directly on HTML documents.
+
+Bridgetown doesn't directly install the Nokogiri gem, so be sure to run `bundle add nokogiri` if it isn't already part of your bundle.
+
+## Your First Inspector
+
+Let's add a oft-requested feature to our site: automatic `target="_blank"` attributes on all outgoing links. It's easy with an HTML inspector.
```

Comment:
```suggestion
Let's add an oft-requested feature to our site: automatic `target="_blank"` attributes on all outgoing links. It's easy with an HTML inspector.
```

Side note, should `Inspector` be capitalized? My thinking here is that `HTML inspector` is not something you can google for (idts anyways after a quick search), so there may be value in signifying it is a noun related to a feature in this project and not an HTML spec.